push_dialog.py

```diff
@@ -24,32 +24,66 @@
 from __future__ import absolute_import
 
 import os
-from PyQt4 import QtGui, uic
-from qgis.core import QgsMapLayerRegistry, QgsProject
+
+from PyQt4 import QtGui, QtCore, uic
 from PyQt4.QtGui import QDialogButtonBox, QPushButton
-from .utils.usb import detect_devices, connect_device, push_file, \
-    disconnect_device
+from qgis.gui import QgsMessageBar
+
+from .export_offline import offline_convert, get_layer_ids_to_offline_convert
+from .data_source_utils import *
+from .config import *
+from .file_utils import fileparts, get_full_parent_path
+from .qt_utils import make_folder_selector
+from .qgis_utils import get_project_title
+
+try:
+    from .utils.usb import detect_devices, connect_device, push_file, \
+        disconnect_device
+except:
+    from mock import MagicMock
```

Comment:
I am just trying to 'mock' the non existent functionality here - is the utils.usb part relevant now? Is it going to be written by you at some point? @mbernasocchi